controllers/discovery/initDiscoveryUserData.js

```diff
@@ -125,7 +126,8 @@ const InitDiscoveryUserData = async (req, res) => {
         allow_anon_dm,
         limit: limit,
         offset: page * limit,
-        minimumKarmaScore: MINIMUM_KARMA_SCORE
+        minimumKarmaScore: MINIMUM_KARMA_SCORE,
+        admin_user_id: process.env.BETTER_ADMIN_ID
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is fine as it passes the `BETTER_ADMIN_ID` environment variable to the query parameters. However, similar to the previous comment, you should handle the case where the environment variable is not set.

```diff
- admin_user_id: process.env.BETTER_ADMIN_ID
+ admin_user_id: process.env.BETTER_ADMIN_ID || 'default_admin_id'
```
Again, replace `'default_admin_id'` with a valid default admin ID.


<!-- This is an auto-generated comment by OSS CodeRabbit -->